DreamCalendar/DreamCalendar/Scene/ScheduleAddition/ScheduleAdditionViewModel.swift

```diff
@@ -17,7 +17,12 @@ final class ScheduleAdditionViewModel: ObservableObject {
     private let delegate: AdditionViewPresentDelegate & RefreshMainViewDelegate
     private let scheduleManager: ScheduleManager
     private var cancellables: Set<AnyCancellable> = []
+<<<<<<< Updated upstream
     private let mode: Mode
+    let tags: [Tag]
+=======
+    private var mode: Mode
+>>>>>>> Stashed changes
```

Comment:
```suggestion
    let tags: [Tag]
    private var mode: Mode
```

`<<<..>>>` 주석 삭제하도록 수정 필요